### PR TITLE
Node crash after moving to iotex-election 0.1.10 #1194

### PR DESCRIPTION
**Pull request by lzxm160**
work on #1194 

According to https://github.com/iotexproject/iotex-core/pull/1210#pullrequestreview-242851417

First fixs poll protocol initializes.

Tests in my local computer shows that wait for about 6 minutes util  initialize ok

### CODE REVIEW

File: action/protocol/poll/protocol.go [hunk 1]
```diff
@@ -197,6 +198,24 @@ func (p *governanceChainCommitteeProtocol) Initialize(
 	log.L().Info("Initialize poll protocol", zap.Uint64("height", p.initGravityChainHeight))
 	var ds state.CandidateList
 	if ds, err = p.delegatesByGravityChainHeight(p.initGravityChainHeight); err != nil {
+		errMsg := "bucket = electionNS doesn't exist: not exist in DB"
```
Comment (zjshen14) [inline action/protocol/poll/protocol.go]: Don't check errorMsg string, but error type.

File: action/protocol/poll/protocol.go [hunk 2]
```diff
@@ -197,6 +198,24 @@ func (p *governanceChainCommitteeProtocol) Initialize(
...
+		errChan := make(chan error, 1)
+		errChan <- err
+	loop:
```
Comment (zjshen14) [inline action/protocol/poll/protocol.go]: No need to have loop label

File: action/protocol/poll/protocol.go [hunk 3]
```diff
@@ -197,6 +198,24 @@ func (p *governanceChainCommitteeProtocol) Initialize(
 	log.L().Info("Initialize poll protocol", zap.Uint64("height", p.initGravityChainHeight))
 	var ds state.CandidateList
 	if ds, err = p.delegatesByGravityChainHeight(p.initGravityChainHeight); err != nil {
+		errMsg := "bucket = electionNS doesn't exist: not exist in DB"
+		errChan := make(chan error, 1)
+		errChan <- err
```
Comment (zjshen14) [inline action/protocol/poll/protocol.go]: You don't have to check error asynchronously. All could happen in a synchronous loop with sleep backoff.

File: action/protocol/poll/protocol.go [hunk 4]
```diff
@@ -197,6 +198,24 @@ func (p *governanceChainCommitteeProtocol) Initialize(
...
+	loop:
+		for {
+			select {
+			case <-time.After(time.Second * 15):
```
Comment (zjshen14) [inline action/protocol/poll/protocol.go]: backoff 15 sec is too conservative, and can we make it configurable

File: action/protocol/poll/protocol.go (hunk 3 above)
Comment (lzxm160) [reply action/protocol/poll/protocol.go]: sure

File: action/protocol/poll/protocol.go (hunk 4 above)
Comment (lzxm160) [reply action/protocol/poll/protocol.go]: ok

**Pull request closed without merging**